WMFModel/WMFArticle+Extensions.h

```diff
@@ -0,0 +1,13 @@
+#import "WMFArticle+CoreDataClass.h"
+
+@interface WMFArticle (WMFExtensions)
+
+@property (nonatomic, readonly, nullable) NSURL *URL;
```

Comment:
looks like you already answered my question…

Only issue I see is that its not clear what "key" is. How does the developer know how to key these?

Maybe add a method like `setKeyWithURL:(NSURL*)url`?